lms/templates/shoppingcart/registration_code_receipt.html

```diff
@@ -37,11 +40,14 @@ <h1 class="title">
         <hr>
         <div>
           % if reg_code_already_redeemed:
-          <% dashboard_url = reverse('dashboard')%>
           <p class="enrollment-text">
             ${_("You've clicked a link for an enrollment code that has already been used."
-            " Check your <a href={dashboard_url}>course dashboard</a> to see if you're enrolled in the course,"
-            " or contact your company's administrator.").format(dashboard_url=dashboard_url)}
+            " Check your {link_start}course dashboard{link_end} to see if you're enrolled in the course,"
+            " or contact your company's administrator."
+            ).format(
+              link_start='<a href="{url}">'.format(url=reverse('dashboard')),
```

Comment:
Please be sure to make this `u""` just in the small offchance that reverse call returns unicode.